From: Adriaan Schmidt <adriaan.schmidt@siemens.com>
Date: Mon, 4 Apr 2022 13:38:33 +0200
Subject: tools: install perf python bindings
Bug-Debian: http://bugs.debian.org/860957
Forwarded: not-needed

---
 tools/perf/Makefile.perf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-master/tools/perf/Makefile.perf
===================================================================
--- linux-master.orig/tools/perf/Makefile.perf	2026-08-21 15:19:58.891258637 +0200
+++ linux-master/tools/perf/Makefile.perf	2026-08-21 15:19:58.890421721 +0200
@@ -908,7 +908,7 @@ install-build-deps:
 	$(Q)$(SHELL) $(srctree)/tools/perf/scripts/install-build-deps.sh $(INSTALL_BUILD_DEPS_ARGS)
 
 install-python_ext:
-	$(PYTHON_WORD) util/setup.py $(python_setup_quiet) install --root='/$(DESTDIR_SQ)'
+	$(PYTHON_WORD) util/setup.py $(python_setup_quiet) install --root='/$(DESTDIR_SQ)' --install-layout=deb
 
 # 'make install-doc' should call 'make -C Documentation install'
 $(INSTALL_DOC_TARGETS):
